lib/fog/kubevirt/compute/models/services.rb

```diff
@@ -0,0 +1,81 @@
+require 'fog/core/collection'
+require 'fog/kubevirt/compute/models/service'
+
+module Fog
+  module Kubevirt
+    class Compute
+      class Services < Fog::Collection
+        attr_reader :kind, :resource_version
+
+        model Fog::Kubevirt::Compute::Service
+
+        def all(filters = {})
+          begin
+            svrs = service.list_services(filters)
+          rescue ::Fog::Kubevirt::Errors::ClientError
+            # we assume that we get 404
+            svrs = []
+          end
+          @kind = svrs.kind
+          @resource_version = svrs.resource_version
+          load svrs
+        end
+
+        def get(name)
+          new service.get_service(name)
+        end
+
+        # Creates a service using provided paramters:
+        # :name [String] - name of a service
+        # :port [int] - a port which will be exposed on node and cluster
+        # :target_port [int] - a vmi port which will be forwarded
+        # :vmi_name [String] - name of a vmi to be selected
+        #
+        # @param [Hash] attributes containing details about service to be
+        #   created.
+        def create(args = {})
```

Comment:
what do you think of adding another util method that would be named create_ssh_service() and will have most of the default values for enabling ssh ? Do you think it will be useful ?

Reply:
Here is how I use it:
> kubevirt.services.create(port: port, name: vm_name, target_port: 22, vmi_name: vm_name)

The only specific value would be `target_port=22` do you think it would be a value to have specific method for it?

Reply:
nope, I assume the service name should be provided as well, i.e. 'ssh' and also the port.
without it, we can leave it as is